keys/types.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Coinbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keys
+
+import "github.com/coinbase/rosetta-sdk-go/types"
+
+// PrivateKey contains the hex-encoded privkey bytes as well as the CurveType
+type PrivateKey struct {
+	HexBytes  string
```

Comment:
Why are we storing strings in the struct instead of bytes in the SDK? It seems preferable to store them in bytes sine Go supports it and only do the hex conversion at the API layer.

It would also help prevent all this extra decoding you're having to do to use the libraries 😏